Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM yastdevel/storage-ng
+RUN zypper --gpg-auto-import-keys --non-interactive in yast2-partitioner
```

Comment:
Actually that's a good thing to do when we go mainline with storage-ng. For now I'd just wait and see.